src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,18 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
-
+    
+   
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */    
+    public String getPrintableString(Printable...printables){
+    	String stringToPrint = "";
+    	
+    	for(Printable printable: printables){
```

Comment:
Is it that there has to be a space between for and (Printable...) 

For e.g.

for (Printable printable: printables) {

?